static/src/javascripts/projects/common/modules/audio/index.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import {
+    Component,
+    React,
+    render,
+} from '@guardian/dotcom-rendering/packages/guui';
+import ophan from 'ophan/ng';
+import Player from './AudioPlayer';
+
+type Props = {
+    source: string,
+    mediaId: string,
+    downloadUrl: string,
+    iTunesUrl: string,
+    pillar: string,
+};
+
+class AudioContainer extends Component<Props, *> {
+    render() {
+        return (
+            <Player
+                sourceUrl={this.props.source}
+                mediaId={this.props.mediaId}
+                downloadUrl={this.props.downloadUrl}
+                iTunesUrl={this.props.iTunesUrl}
+                barWidth={2}
+                controls="controls"
+                pillar={this.props.pillar}
+            />
+        );
+    }
+}
+
+const sendToOphan = () => {
+    ophan.record({
+        componentEvent: {
```

Comment:
I thought we were tracking this with data-component and the standard media events?

Reply:
riight yes so we don't need the old code.

Reply:
I think this branch has lost the code we wrote for sending the audio events?
e.g.
```
audio: {
  id: "gu-audio-5af940e8e4b02d1f3350ef21",
  eventType:"audio:content:play"
}
```
and one for "audio:content:ready"

Reply:
🙈 aw no. It must have got overwritten in one of the many rebases